### ISSUE TITLE
NPC vender new prices?!

### ISSUE THREAD

**Issue by AzarthMZintos** [rathena/rathena#2770]
* **rAthena Hash**: 

* **Client Date**: 

* **Server Mode**: 

* **Description of Issue**: 
I was looking at divine-pride, at npc prices and alot of them change: like item fly wing(id:601) in the rathena is 60z(npc vender) in the divine pride is 250z, the item butterfly wing(id:602) rathena = 300z, divine pride is 1000z.
Anyone know is those divine-pride is KRO official "new" prices?, is so rathena will update those prices, problably there is more itens, anyway thats it.
  * Result: 
  * Expected Result: 
  * How to Reproduce: 
  * Official Information:
  

* **Modifications that may affect results**:

**Comment by AzarthMZintos** [comment]
The ones I see so far AGAIN I don't play official servers so if someone could check will be better and the sell prices of some change too if anyone wanna check it too =)

prices change based on regular npcs:
579 = Fresh Fish // rathena price: 250 //divine npc price: 700
601 = Fly Wing // rathena price: 60 //divine npc price: 250
602 = Butterfly Wing // rathena price: 300 //divine npc price: 1000
610 = Yggdrasil Leaf // rathena price: 4,000 // divine npc price: 12,000
611 = Magnifier // rathena price: 40 // divine npc price: 200
643 = Pet Incubator // rathena price: 3,000 // divine npc price: 10,000
645 = Concentration Potion // rathena price: 800 // divine npc price: 1,200
656 = Awakening Potion // rathena price: 1,500 // divine npc price: 2,200
657 = Berserk Potion // rathena price: 3,000 // divine npc price: 4,500

prices change based on paramarket npcs:

505 = Blue Potion // rathena price: 5,000 // divine npc price: 40,000
525 = Panacea // rathena price: 500 // divine npc price: 100,000
526 = Royal Jelly // rathena price: 7,000 // divine npc price: 50,000
576 = Prickly Fruit // rathena price: 540 // divine npc price: 16,500
587 = Red Prickly Fruit // rathena price: 880 // divine npc price: 19,800
607 = Yggdrasil Berry // rathena price: 5,000 // divine npc price: 200,000
616 = Old Card Album // rathena price: 10,000 // divine npc price: 2,000,000

**Comment by secretdataz** [comment]
Your list isn't exhaustive. We can consider issues related to buy/sell price changes related to this issue.

**Comment by Lemongrass3110** [comment]
You can simply comment your list into the other issue. It is just hard to follow these kind of information if they are split into many issues. This could end up in one issue per item which would be overkill. :-)

**Comment by zackdreaver** [comment]
Those are from Free server aka sakray of kRO,
The prices are normal in premium server.

I remember I have discussed this issue somewhere around 2014/2015.

**Comment by AzarthMZintos** [comment]
Any updates?
